__tests__/app.test.js

```diff
@@ -36,10 +49,40 @@ afterAll(() => {
         return request(app)
         .get("/api/reviews")
         .expect(200)
-        .then(({})=>{
-
-
+        .then(({body})=>{
+            const { reviews } = body;
+            expect(reviews.length).toBe(13)
+            expect(reviews).toBeInstanceOf(Array)
```

Comment:
We wouldn't need line 55 in this case, as we already know it's an array later by using forEach. You can therefore remove line 55 👍